packages/server/lib/environment.coffee

```diff
@@ -19,6 +19,10 @@ try
   app = require("electron").app
   app.commandLine.appendSwitch("disable-renderer-backgrounding", true)
   app.commandLine.appendSwitch("ignore-certificate-errors", true)
+  ## this should really only be necessary when 
+  ## running Chromium versions >= 72
+  ## https://github.com/cypress-io/cypress/issues/1872
+  app.commandLine.appendSwitch("proxy-bypass-list", "<-loopback>")
```

Comment:
this definitely does not need to be here. 

we already handle proxy rules during `BrowserWindow` creation. 

the only top level arguments that have to be passed to electron are ones that cannot be passed at a later time during `BrowserWindow` construction.

Reply:
We cannot use this command line flag because it would affect our own host electron version instead of affecting just the instantiated BrowserWindow instance we use for testing user's application.

We simply need to bypass the proxy using the `session` object we already use when we construct the Electron browser window. Per the docs this option is programmatically controllable as documented here: https://electronjs.org/docs/api/session#sessetproxyconfig-callback

This PR needs to utilize this option and update the electron tests and then we're good to merge.

Reply:
We cannot use this command line flag because it would affect our own host electron version instead of affecting just the instantiated BrowserWindow instance we use for testing user's application.

We simply need to bypass the proxy using the session object we already use when we construct the Electron browser window. Per the docs this option is programmatically controllable as documented here: https://electronjs.org/docs/api/session#sessetproxyconfig-callback

This PR needs to utilize this option and update the electron tests and then we're good to merge.

Reply:
I will update, but why if this implementation is problematic, why didn't any of our tests fail? Shouldn't we have some failing test?